liver_imaging_analysis/engine/engine.py

```diff
@@ -135,12 +149,54 @@ def get_metrics(self, metrics_name, **kwargs):
                 parameters of metrics, if exist.
         """
         metrics = {
-            "accuracy" : Accuracy,
-            "dice" : DiceMetric,
-            "jaccard" : MeanIoU,
+            "accuracy": Accuracy,
+            "dice": DiceMetric,
+            "jaccard": MeanIoU,
         }
         return metrics[metrics_name](**kwargs)
 
+    def get_hparams(self, network_name):
+        """
+        used to return a hyperparameters dictionary specific to each network
+        ----------
+            network_name: str
+                name of network to fetch from dictionary
+                should be chosen from: '3DUNet','3DResNet','2DUNet'
+        """
+        hparams = {
+            "monai_2DUNet": {
```

Comment:
where are the other networks? Is this function supposed to be an engine method? Can it be more generic? I guess this will be messy when adding other networks.

Reply:
you are right but it also depends on network parameters names which are different